src/app/connections/list-toolbar/list-toolbar.component.html

```diff
@@ -45,7 +45,7 @@
           </button>
       </div>
       <div class="toolbar-pf-action-right">
-        <button *ngIf="showCreate" type="button" class="btn btn-primary" [routerLink]="['create']">Create
+        <button *ngIf="showCreate" type="button" class="btn btn-primary" [routerLink]="['/connections/create']">Create
```

Comment:
However I don't mind changing it, just wanted to let you know it seems to be working fine. Let me know if you'd like for me to change it.